packages/terra-date-input/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 * Added
   * Added hot keys functionality to support keyboard shortcut patterns.
+  * Added black dashed border when input gets focused.
+  * Added screen reader support for error messages.
```

Comment:
Pls move this entry to unreleased section

Reply:
Updated.